### ISSUE TITLE
cannot create RSA Key in PHP7

### ISSUE THREAD

**Issue by juliussaputra** [Spomky-Labs/jose#329]
while creating RSAKey I got a crashed of my script which runs in a PHP7 environment.
We didn't have this problem with older version of PHP.
We assume it is a namespace problem. 

These are the following stack trace, that we got on our script:

``` 
line 39 of /auth/swisseduid/vendor/spomky-labs/jose/src/Util/BigInteger.php: Error thrown
line 64 of /auth/swisseduid/vendor/spomky-labs/jose/src/Util/BigInteger.php: call to Jose\Util\BigInteger->__construct()
line 400 of /auth/swisseduid/vendor/spomky-labs/jose/src/KeyConverter/RSAKey.php: call to Jose\Util\BigInteger::createFromBinaryString()
line 372 of /auth/swisseduid/vendor/spomky-labs/jose/src/KeyConverter/RSAKey.php: call to Jose\KeyConverter\RSAKey->convertBase64StringToBigInteger()
line 84 of /auth/swisseduid/vendor/spomky-labs/jose/src/KeyConverter/RSAKey.php: call to Jose\KeyConverter\RSAKey->populateBigIntegers()
line 107 of /auth/swisseduid/vendor/spomky-labs/jose/src/Factory/JWKFactory.php: call to Jose\KeyConverter\RSAKey->__construct()
line 93 of /auth/swisseduid/lib/OAuthManager.php: call to Jose\Factory\JWKFactory::createRSAKey()
```
The error message is

```
Exception - Call to undefined function Jose\Util\gmp_init()
```

**Comment by juliussaputra** [comment]
This is related to #218
we don't think that this is an environment problem, but related to more restrictive syntax requirements in PHP7

**Comment by Spomky** [comment]
Hi,

Can you please try to create a RSA key using the new JWT App?

```sh
curl -OL https://github.com/web-token/jwt-app/raw/gh-pages/jose.phar
curl -OL https://github.com/web-token/jwt-app/raw/gh-pages/jose.phar.pubkey

jose.phar key:create:rsa 2048 # Creates a 2048 bits RSA key
```

**Comment by juliussaputra** [comment]
Hi,

just wanted to report it we fixed the issue already.
We used the framework actually for our Moodle system, namely for the authentication plugin.
The new framework web-token/jwt-app was somehow not compatible with moodle at least on our side.

So the problem was actually the namespace on the jose/src/util/BigInteger
on the constructor function
```
$this->value = \gmp_init($value, $base);
```
and also the gmp extension for php was not installed in our server, so after installed and enabled the gmp on our server php. Everything works again as normally.
